### PR TITLE
Update Arcade TFMs according to TFM docs

### PR DESCRIPTION
**Pull request by ViktorHofer**
- Use "Tool" floating TFM properties for msbuild tasks and tools, i.e. Microsoft.DotNet.Arcade.Sdk.
- Use "Minimum" floating TFM properties for libraries that support previous framework versions, i.e. XUnitExtensions.
- Use "Current" floating TFM properties for libraries that support only the current framework version.

### CODE REVIEW

File: src/Microsoft.DotNet.ArcadeLogging/Microsoft.DotNet.ArcadeLogging.csproj [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <IsPackable>true</IsPackable>
```
Comment (ViktorHofer) [inline src/Microsoft.DotNet.ArcadeLogging/Microsoft.DotNet.ArcadeLogging.csproj:5]: The library was marked as packable but never got consumed anywhere else than directly in Arcade.Sdk.csproj. It doesn't make sense to be packaged up.

File: src/Microsoft.DotNet.Helix/Sdk/tools/Microsoft.DotNet.Helix.Sdk.props [hunk 2]
```diff
@@ -4,13 +4,13 @@
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" />
   <Import Project="$(MSBuildToolsPath)\Microsoft.NETCoreSdk.BundledVersions.props" Condition="Exists('$(MSBuildToolsPath)\Microsoft.NETCoreSdk.BundledVersions.props')" />
 
-  <PropertyGroup Condition="'$(MicrosoftDotNetHelixSdkTasksAssembly)' == ''">
```
Comment (akoeplinger) [inline src/Microsoft.DotNet.Helix/Sdk/tools/Microsoft.DotNet.Helix.Sdk.props:7]: why did you remove this condition? it looks like this is used in tests

**Review by akoeplinger — DISMISSED** [review]

**Review by akoeplinger — APPROVED** [review]
